src/Analysis/Engine/Impl/AnalysisLimits.cs

```diff
@@ -180,6 +180,12 @@ public sealed class AnalysisLimits {
         /// </summary>
         public bool ProcessCustomDecorators { get; set; }
 
+        /// <summary>
+        /// <c>True</c> to propagate parameter types to base methods.
+        /// Parameter types always propagate to derived methods.
+        /// </summary>
+        public bool PropagateParameterTypeToBaseMethods { get; set; }
```

Comment:
Is there case when we don't want this?

Reply:
I mean, it is not something that user sets so it it typically set by the application one way or another. #261 may be related.